.vsts-ci.yml

```diff
@@ -0,0 +1,107 @@
+resources:
+- repo: self
+  clean: true
+queue:
+  name: VSEng-MicroBuildVS2017
+  condition: succeeded()
+  demands: Cmd
+variables:
+  BuildConfiguration: Release
+  TeamName: Roslyn
+  
+steps:
+- task: ms-vseng.MicroBuildTasks.30666190-6959-11e5-9f96-f56098202fef.MicroBuildSigningPlugin@1
+  displayName: Install Signing Plugin
+  inputs:
+    signType: real
+    esrpSigning: true
+    
+- task: ms-vseng.MicroBuildTasks.32f78468-e895-4f47-962c-58a699361df8.MicroBuildSwixPlugin@1
+  displayName: Install Swix Plugin
+  
+- task: NuGetRestore@1
+  displayName: Restore internal toolset
+  inputs:
+    solution: 'build\Internal\Toolset.csproj'
+    feed: '8f470c7e-ac49-4afe-a6ee-cf784e438b93'
+
+- task: BatchScript@1
+  displayName: Run script CIBuild.cmd
+  inputs:
+    filename: 'build\CIBuild.cmd'
+    arguments: '-configuration $(BuildConfiguration)'
+    modifyEnvironment: false
+    
+- task: PublishTestResults@1
+  displayName: Publish Test Results
+  inputs:
+    testRunner: XUnit
+    testResultsFiles: 'artifacts/$(BuildConfiguration)/TestResults/*.xml'
+    testRunTitle: 'Unit Test Results'
```

Comment:
I think this needs `condition: succeededOrFailed()` in order to publish tests if they fail (the build script returns non-zero exit code if tests fail)

Reply:
fixed